libs/utils/src/Card/web_service_card.ts

```diff
@@ -29,8 +37,9 @@ export class WebServiceCard implements Card {
   async readLongObject<T>(
     schema: z.ZodSchema<T>
   ): Promise<Result<Optional<T>, SyntaxError | z.ZodError>> {
-    const response = await fetch('/card/read_long');
-    const { longValue } = await response.json();
+    const { longValue } = (await fetchJson(
+      '/card/read_long'
+    )) as LongValueResponse;
```

Comment:
The `fetchJson` helper checks status codes via the `ok` field on responses (which is set to false for 4xxs and 5xxs):

https://github.com/votingworks/vxsuite/blob/42069db8d35f25a239f0e7880e9d137215e5dbc4/libs/utils/src/fetch_json.ts#L13-L15

The base `fetch` doesn't throw errors for 4xxs and 5xxs. So before this PR, 4xxs and 5xxs would result in us accessing properties of undefined objects, etc.

Reply:
Nice! That seems much better. My only concern is that based on that code snippet you linked, it looks like we don't surface the actual error code or message anywhere.

Reply:
Good point! I updated `fetch_json` to at least include the status code in its error messages: https://github.com/votingworks/vxsuite/pull/2135/commits/72c7b99897f0994bdf99da9c23caf327f584be89